### ISSUE TITLE
ToDo default value dependent on other custom fields

### ISSUE THREAD

**Issue by igor5616377** [AndreyVMarkelov/mailru-jira-plugins-todolist#16]
Приветствую!

Мы исползуем Ваш плагин ToDo list. Плагин работает корректно, огромное спасибо!

Просьба настроить значение по умолчанию с возможностью указания зависимостей от значений других полей.

Например,у меня есть каскадное поле с выбором одного значения 
(пример: [Обновление сервера], [Hастройка транка], и т.д.) 
При выборе определенного значения в этом поле в тикете должно создаваться соответствующий ToDo для данной задачи.
Что-то типа- 
If customfield-123="Обновление сервера" then
ToDo=
1. Проверить доступность сервера
2. Скопировать папку с новой версией на сервер
3. Обновить сервер
4. Проверить работу
5. Обновить версию в вики
5. Залогинить время
6. Закрыть задачу.
else NULL

Возможно ли это? Заранее спасибо!

**Comment by AndreyVMarkelov** [comment]
Добрый день!
Имеется ввиду добавить пост функцию создания значения в ToDO?

**Comment by igor5616377** [comment]
Да, все верно! :) но значение должно быть зависимым от значения другого customfield.

**Comment by igor5616377** [comment]
Просьба сообщить, Вы планируете добавить данный функционал? Возможно подскажете  существующее решение задачи?
